resources/views/checkout.blade.php

```diff
@@ -0,0 +1,27 @@
+<button
+    id="checkout-{{ $sessionId }}"
+    role="link"
+    style="background-color:#6772E5;color:#FFF;padding:8px 12px;border:0;border-radius:4px;font-size:1em"
+>
+    {{ $label }}
+</button>
+
+<div id="error-message"></div>
+
+<script>
+    const checkoutButton = document.getElementById('checkout-{{ $sessionId }}');
```

Comment:
one thing we've run into is a few users will try to put multiple buttons on one page. I wonder if it makes sense to wrap this bit in an iife to keep the scope of `checkoutButton` inside of the single view?

```html
<script>
(() => {
  const checkoutButton = doc...
  //
  
)()
</script>
```